tex/rep-theory/semisimple.tex

```diff
@@ -209,6 +261,43 @@ \section{Density theorem}
 	$\bigoplus_i V_i^{\oplus d_i}$ (by \Cref{cor:subrep_schur})
 	and by the above $d_i = \dim V_i$ for each $i$.
 \end{proof}
+% there's another proof in https://www3.nd.edu/~andyp/notes/JacobsonDensity.pdf
+% which might be more digestible
+
+\begin{example}
+	[Applying the proof of density theorem on an explicit example]
```

Comment:
Honestly, I still don't have a very good intuition on what makes the proof work. It's not possible to run the argument through in cases like A = Mat_2(k) and V = (k^2) \oplus (k^2), or A = some non-semisimple algebra and V = some reducible but indecomposable representation.